communities/plymouth-university/community-assessment.md

```diff
@@ -0,0 +1,122 @@
+
+I'll be using this assessment to investigate the culture and community of Plymouth University as a whole rather than focusing on the GDSoc alone.
+
+## Definition
+
+Students at the University of Plymouth, UK (known as Plymouth University) who have an interest in technology and computer science. These students may be on computing / computer science courses or just have an interest they're looking to explore.
+
+## Size
+
+### How many people are currently in the community?
+
+The size of the current involved community is sitting around **50**. This is just the engaged community who have together to take part and create events for themselves, primarily this is GDSoc along with a number of enthusiastic students.
+
+### How many would you _like_ to be in the community?
+
+In the distant future it would be wonderful to see a of the majority of the School of Computing engaging in the community, this would be around 200.
+
+In a smaller time frame it would be good to get all 50 together for regular meets such as workshops and talks.
+
+### Historically, has the group always been about the same size? If not, what brought on that change?
+
+The University had a lot more tech activity when I was a fresher in 2012. It seems that students are less willing to turn up to events now, the quality seems to have remained the same and even improved yet turn out is much lower now. There is a lot less "buzz" around tech news, new tools, and opportunities.
+
+## Resources
+
+In particular when considering GDSoc, the amount of resources avalible to us are absolutely huge. The rest of the School of Computing also have quite a few resource avalible to us but they are not so "toy-driven".
+
+### What is current available to people in your community to learn about tech?
+
+We probably have one of the biggest VR and "random-peripheral" stores for a University School of Computing in terms of hardware and software. This includes:
+
+* Several HTC Vives
+* Oculus Rifts
+* Leap Motions
+* Google Cardboards and a few other random VR kits
+* Force feedback vests
+* A high spec OSX lab
+* A high spec Windows Lab (GTX 1080s etc)
+
+In terms of locations, and other resources we have access to:
+
+* Lecture theatres and room bookins (under a society or via a staff member)
+* Projectors and screens for talks
+* Flat screen TV's for smaller presentations
+* GoPro's for filming events
+
+We have a lot on offer for people to take and hack around with equipment. They can be booked out and used for projects, events, and demos as students wish. The place we're lacking is with events and knowledge spreading. A common issue seems to be that someone learns something but doesn't feel the need to share it, they either assume it's obvious, unimportant, or never consider sharing it.
```

Comment:
> they either assume it's obvious, unimportant, or never consider sharing it. 

There's another side to this. People tend to vastly underestimate the value of their own experiences. A hugely valuable function you could provide here is communicating to people the value of their knowledge and avenues to share it.